shared_model/backend/protobuf/queries/proto_query.hpp

```diff
@@ -142,6 +142,11 @@ namespace shared_model {
         return true;
       }
 
+      bool clearSignatures() override {
```

Comment:
Query set has only one signature.
Make sure it is cleared - may be add test for it?

Reply:
Not the scope of this PR